sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobAsyncClient.java

```diff
@@ -82,6 +83,7 @@
  * <a href=https://docs.microsoft.com/rest/api/storageservices/understanding-block-blobs--append-blobs--and-page-blobs>Azure
  * Docs</a> for more information.
  */
+@ServiceClient(builder = BlobClientBuilder.class, isAsync = true)
```

Comment:
Are we updating this to keep it up with guidelines? Same for other Storage clients.

Reply:
Yes, this is a service client but was missing the annotation.